sql/moz-fx-data-shared-prod/app_store_syndicate/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: app_store_syndicate
+description: |
+  Syndicated App Store data from Fivetran. User-facing views into this data can be found
```

Comment:
I'm guessing the part about `ads` is copypasta and should be removed.

Reply:
yes, thanks for catching this.